nixos/tests/gitea.nix

```
@@ -19,6 +20,14 @@ let
           enable = true;
           database = { inherit type; };
           disableRegistration = true;
+          cache = if (cache == "redis") then {
+            redis.enable = true;
+          } else if (cache == "memcache") then {
+            memcached.enable = true;
+          } else if (cache == "memory") then {
+            memory.enable = true;
+          } else {
+          };
```

```suggestion
          cache = { ${cache}.enable = true; };
```

This actually works.  If `cache` is `null` this will evaluate to an empty attribute set:
```
nix-repl> cache = null

nix-repl> { ${cache}.enable = true; }
{ }
```